CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 **********
 
+[1.4.0] - 2022-10-21
+********************
+
+Changed
+=======
+* Remove auto.offset.reset: earliest from consumer (new consumer groups will now consume only messages that are sent after the group was initialized)
```

Comment:
Adjusted language to match the net result, rather than the fact that you technically removed a line.
```suggestion
* Changed "auto.offset.reset" from "earliest" to "latest" (the default value). New consumer groups will now consume only messages that are sent after the group was initialized.
```